api/src/entities/collections.rs

```diff
@@ -33,6 +33,8 @@ pub enum Relation {
     CollectionMints,
     #[sea_orm(has_one = "super::drops::Entity")]
     Drop,
+    #[sea_orm(has_many = "super::mint_history::Entity")]
+    MintHistory,
```

Comment:
```suggestion
    MintHistories,
```

plural since has many mint_history records